composer.json

```diff
@@ -8,8 +8,8 @@
         }
     ],
     "require": {
-        "php": ">=7.0.0",
-        "phpunit/phpunit": "^6.5"
```

Comment:
You could instead to `"phpunit/phpunit": "^6.5||^8.5"` as alot of other libraries do to allow composer to select the compatible package.

Reply:
But why? 8.5 is already ancient. This should be `^9` already.

Reply:
Probably, however they is more likely to be changes to tests going up to PHPUnit 9.5. Where as 8.5 supports php >=7.2, so it will install and run tests for PHP 8 & 8.1, which I think is the main issue with getting the extension to pass the CI tests on php 8+

Reply:
PHP 7.2 is EOLed already so there's no point supporting it.